README.md

```diff
@@ -83,25 +83,40 @@ _Video.js uses [BrowserStack][browserstack] for compatibility testing._
 
 Video.js is [licensed][license] under the Apache License, Version 2.0.
 
-
 [browserstack]: https://browserstack.com
-[buildwith]: https://trends.builtwith.com/media/VideoJS
+
```

Comment:
Doesn't seem like there's an option for remark for this right now. I'll keep my eye on it.